gtest/src/manager/exec.rs

```diff
@@ -35,6 +35,33 @@ impl ExtManager {
         self.route_dispatch(dispatch)
     }
 
+    pub(crate) fn claim_value_impl(
```

Comment:
by your choice: I'd move it to `mailbox::actor` too

Reply:
Fixed